frontend/packages/dev-console/src/components/AddPage.tsx

```diff
@@ -1,17 +1,49 @@
 import * as React from 'react';
 import { Helmet } from 'react-helmet';
+import { match as RMatch } from 'react-router';
+import { history } from '@console/internal/components/utils';
+import { createProjectModal } from '@console/internal/components/modals';
 import ODCEmptyState from './EmptyState';
 import NamespacedPage from './NamespacedPage';
+import DefaultPage from './DefaultPage';
 
-const AddPage: React.FC = () => (
-  <React.Fragment>
-    <Helmet>
-      <title>+Add</title>
-    </Helmet>
-    <NamespacedPage>
-      <ODCEmptyState title="+Add" />
-    </NamespacedPage>
-  </React.Fragment>
-);
+export interface AddPageProps {
+  match: RMatch<{
+    ns?: string;
+  }>;
+}
+const openProjectModal = () =>
+  createProjectModal({
+    blocking: true,
+    onSubmit: (project) => history.push(`/add/ns/${project.metadata.name}`),
+  });
+const AddPage: React.FC<AddPageProps> = ({ match }) => {
+  const namespace = match.params.ns;
+  return (
+    <React.Fragment>
+      <Helmet>
+        <title>+Add</title>
+      </Helmet>
+      <NamespacedPage>
+        {namespace ? (
+          <ODCEmptyState title="+Add" />
+        ) : (
+          <DefaultPage title="+Add">
```

Comment:
@serenamarie125 Please confirm if the page header should be `+Add` or `Add`

Reply:
Page header should just be “Add”